app/core/model/account/errors.go

```diff
@@ -0,0 +1,24 @@
+package account
+
+import "errors"
+
+var (
+
+	// ErrCreating occurs when an untracked error happens when creating a new account
+	ErrCreating = errors.New("an error occurred while creating this account")
+
+	// ErrInvalid throws when the identification provided is an invalid form of ID
+	ErrInvalid = errors.New("the account id provided is invalid. please, try again with a valid one")
+
+	// ErrNotFound expresses that the account with the identification provided does not exist, or, for some reason, could not be found
+	ErrNotFound = errors.New("the account requested could not be found or does not exist")
+
+	// ErrExists points out that an account with the CPF provided in the request, which is unique, already exists in the database
+	ErrExists = errors.New("an account with the document provided already exists")
```

Comment:
não sei, mas acho que fica mais claro dessa forma do que se trata o erro, o que vc acha?

```suggestion
	ErrAlreadyExists = errors.New("an account with the document provided already exists")
```

Reply:
Sugestão atendida! Também acho que fica mais explicito com o already